ext/opcache/tests/gh8065.phpt

```diff
@@ -0,0 +1,22 @@
+--TEST--
+Bug GH-8065: Opcache zend_class_entry.inheritance_cache breaks persistent script checksum
+--INI--
+opcache.enable=1
+opcache.enable_cli=1
+opcache.consistency_checks=1
+opcache.log_verbosity_level=2
+opcache.protect_memory=1
+--EXTENSIONS--
+opcache
+--FILE--
+<?php
+
+class Foo {}
+
+require_once __DIR__ . '/gh8065.inc';
+
+echo "Done\n";
+
+?>
+--EXPECT--
+Done
```

Comment:
The test passes without patch as well.

Reply:
This is because this PR changes the level of the given message.

https://github.com/php/php-src/pull/10624/files#diff-b66a794224587a12f8650298f931f4feafb7734f00885ad885f609a68e840949R2123